tensorflow/core/kernels/mkl/mkl_qmatmul_op.cc

```diff
@@ -436,13 +476,36 @@ class MklDnnQuantizedMatMulOp : public MklDnnMatMulOpBase<Tweight, Toutput> {
       output_scale.push_back(scale);
       params.post_op_params.push_back({"output_scale", output_scale});
     }
+#else
+      if (std::is_same<Toutput, quint8>::value) {
+        // Existing unit tests expect dst_scale to be computed using 256.0
```

Comment:
Are these unit-tests oneDNN-specific? If so, we probably should just change the tests instead.

Reply:
Yes. Fixed the test.